fetcher/src/job/job.processor.ts

```diff
@@ -47,14 +43,18 @@ export class JobProcessor extends WorkerHost {
           value: aggregate
         })
         this.logger.debug(response)
-        const outData: IDeviationData = {
-          timestamp: timestamp,
-          submission: aggregate,
-          oracleAddress
-        }
-        if (
-          shouldReport(Number(lastSubmission), aggregate, decimals, threshold, absoluteThreshold)
-        ) {
+
+        const contract = new ethers.Contract(oracleAddress, Aggregator__factory.abi, PROVIDER)
```

Comment:
@bayram98 It would be better to create the `contract` (as a map) somewhere else so we do not have to recreate it every time we fetch new data every two seconds. Please suggest a solution or at least open an issue for it.

Reply:
@martinkersner That is really good point. I will work on it. Maybe we could read all the contracts once the fetchers are activated, and reuse it.

Reply:
You should make sure it is also updated when new fetchers are activated.